odoo/tools/paper_muncher.py

```diff
@@ -0,0 +1,128 @@
+import logging
+import subprocess
+
+from werkzeug.test import create_environ, run_wsgi_app
+
+from odoo.http import request, root, Session
+
+from .misc import find_in_path
+
+_logger = logging.getLogger(__name__)
+
+REPORT_ASSET_KEY = 'http://127.0.0.1:8069/papermuncher0000.html'
+
+
+def can_use_papermuncher():
+    if not request or 'use-papermuncher' not in request.session.debug:
+        return False
```

Comment:
there's a way of doing undocummented settings in the configuration file (the `--config`/`.odoorc`) we could support a `papermuncher=1` in the settings that would always attempt to use papermuncher, ignoring the query-string param

tell me if you wanna do that